shared/constants/countries.ts

```diff
@@ -1,4 +1,5 @@
 export enum Country {
+  Singapore = 'SINGAPORE',
```

Comment:
Hmm, I'm not sure we should do that 🤔. It's a quick hack, but it looks like a breach of concern.

The `Country` enum is a shared constant, and it's raw data. Changing the key order to achieve a rendering objective in one component specifically (the CountryField) is an overreach.

Presenting Singapore first in the country field is a display concern, and should probably be done in the `CountryField` component itself instead. There'll be a bit more code to achieve that goal, but it'll be more explicit, and `CountryField` could take a `defaultCountry` arg, that defaults to Singapore.

Also can I clarify: is the requirement to have Singapore **first** in the list, or is it to have Singapore **preselected** in the list (i.e. default value)? If the later, we can use the `selected` attribute on the option to preselect it, while maintaining the overall country list order.

Reply:
Will be resolving this thread as the necessary refactor has been done. Thank you!